src/containers/Login.js

```diff
@@ -1,17 +1,37 @@
 import React, { useState } from "react";
 import { Button, FormGroup, FormControl, ControlLabel } from "react-bootstrap";
 import "./Login.css";
+import Credentials from "../data/credentials.json";
 
 export default function Login(props) {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  var userFound = "no";
 
   function validateForm() {
     return email.length > 0 && password.length > 0;
   }
 
-  function handleSubmit(event) {
-    event.preventDefault();
+  async function handleSubmit(event) {
+    try {
+      if (email && password) {
+        Credentials.map((credentialData, index) => {
+          if (credentialData.email === email && credentialData.password === password) {
+            event.preventDefault();
+            userFound = "yes";
+            alert("Logged in");
```

Comment:
Not alert we will use a new page may be empty page

Reply:
Did this intentionally so that users remain on the same page. Handling an alert or pop-up looks easier than going to a different page and clicking the back button.